libs/features/personalization/personalization.js

```diff
@@ -124,35 +124,44 @@ const createFrag = (el, url, manifestId) => {
   return frag;
 };
 
+const GLOBAL_CMDS = [
+  'insertscript',
+  'replacepage',
+  'updatemetadata',
+  'useblockcode',
+];
+
+const CREATE_CMDS = {
+  insertafter: 'afterend',
+  insertbefore: 'beforebegin',
+  prependtosection: 'afterbegin',
+  appendtosection: 'beforeend',
+};
+
 const COMMANDS = {
-  insertcontentafter: (el, target, manifestId) => el
-    .insertAdjacentElement('afterend', createFrag(el, target, manifestId)),
-  insertcontentbefore: (el, target, manifestId) => el
-    .insertAdjacentElement('beforebegin', createFrag(el, target, manifestId)),
-  removecontent: (el, target, manifestId) => {
+  remove: (el, target, manifestId) => {
     if (target === 'false') return;
     if (manifestId) {
       el.dataset.removedManifestId = manifestId;
       return;
     }
     el.classList.add(CLASS_EL_DELETE);
   },
-  replacecontent: (el, target, manifestId) => {
-    if (el.classList.contains(CLASS_EL_REPLACE)) return;
+  replace: (el, target, manifestId) => {
+    if (!el || el.classList.contains(CLASS_EL_REPLACE)) return;
     el.insertAdjacentElement('beforebegin', createFrag(el, target, manifestId));
     el.classList.add(CLASS_EL_DELETE, CLASS_EL_REPLACE);
   },
 };
 
-const VALID_COMMANDS = Object.keys(COMMANDS);
-
-const GLOBAL_CMDS = [
-  'insertscript',
-  'replacefragment',
-  'replacepage',
-  'updatemetadata',
-  'useblockcode',
-];
+function checkSelectorType(selector) {
+  let selectorType = 'css';
+  if (selector.includes('/fragments/')
+    && (selector.startsWith('/') || selector.startsWith('http'))) {
+    selectorType = 'fragment';
+  }
+  return selectorType;
+}
```

Comment:
CSS selector would not have slashes, so, imho, checking for '/fragments/' should be enough.
Suggestion:
```
const selectorType = (selector) => selector?.includes('/fragments/') ? 'fragment' : 'css';
```

i

Reply:
You're right, I was probably being overly cautious. Updated to ternary.
```
function checkSelectorType(selector) {
  return selector?.includes('/fragments/') ? 'fragment' : 'css';
}
```